bugbot/rules/uplift_beta.py

```diff
@@ -105,30 +133,28 @@ def get_bz_params(self, date):
             "f3": "flagtypes.name",
             "o3": "notsubstring",
             "v3": "approval-mozilla-beta",
-            "f4": "flagtypes.name",
-            "o4": "notsubstring",
-            "v4": "needinfo",
             # Don't nag several times
-            "n5": 1,
-            "f5": "longdesc",
-            "o5": "casesubstring",
+            "n4": 1,
+            "f4": "longdesc",
+            "o4": "casesubstring",
             # this a part of the comment we've in templates/uplift_beta_needinfo.txt
-            "v5": ", is this bug important enough to require an uplift?",
+            "v4": ", is this bug important enough to require an uplift?",
             # Check if have at least one attachment which is a Phabricator request
-            "f6": "attachments.mimetype",
-            "o6": "anyexact",
-            "v6": ["text/x-phabricator-request", "text/x-github-pull-request"],
+            "f5": "attachments.mimetype",
+            "o5": "anyexact",
+            "v5": ["text/x-phabricator-request", "text/x-github-pull-request"],
             # skip if whiteboard contains checkin-needed-beta (e.g. test-only uplift)
-            "f7": "status_whiteboard",
-            "o7": "notsubstring",
-            "v7": "[checkin-needed-beta]",
+            "f6": "status_whiteboard",
+            "o6": "notsubstring",
+            "v6": "[checkin-needed-beta]",
```

Comment:
nit: I would keep the old number to preserve a clean blame history.

Reply:
495955bbe112d9c806586e4365226cebb6a73889